code/src/Select.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import "./select.css"
+
+export const Select = ({ vacation, setVacation }) => (
+  <select
+    onChange={event => setVacation(event.target.value)}
+    value={vacation}
+    required
+  >
+    <option value="">Select your dream vacation:</option>
```

Comment:
Guess you don't want this to be an option? :)
```suggestion
    <option value="" disabled>Select your dream vacation:</option>
```